### PR TITLE
Prep for v0.16.0

### PR DESCRIPTION
**Pull request by odow**
Approved by:

- [x] @ericphanson 
- [x] @blegat

### CODE REVIEW

**Review by blegat — APPROVED** [review]

**Review by ericphanson — APPROVED** [review]

File: docs/src/changelog.md [hunk 1]
```diff
@@ -40,7 +44,7 @@ changes.
    but it can result in subtle differences, particularly for code like
    `x[i, :] * y[i, :]'`: this used to be equivalent to the inner product, but it
    is now the outer product. In Base Julia, this is the outer product, so the
-   previous code may be been silently broken (#624)
+   previous code may have be been silently broken (#624)
```
Comment (ericphanson) [inline docs/src/changelog.md:47]:
```suggestion
   previous code may have been silently broken (#624)
```

**Comment by odow** [comment]
I'll wait for @blegat to wake up to confirm

**Comment by ericphanson** [comment]
I found two new issues #683 and #682 but neither are regressions from v0.15.4 so I think they can be fixed in a patch release post-v0.16.

**Comment by odow** [comment]
I'm not going to wait for #685. I'll merge and tag this once CI passes.

**Pull request merged**